src/main/java/org/beanpod/switchboard/controller/LogController.java

```diff
@@ -60,8 +65,9 @@
   @Override
   public ResponseEntity<StreamLogModel> createStreamLog(
       CreateStreamLogRequest createStreamLogRequest) {
+    UserEntity user = userDao.findUser(request.getUserPrincipal().getName());
     return Optional.of(createStreamLogRequest)
-        .map(streamLogService::createLog)
+        .map(createStreamLogReq -> streamLogService.createLog(user, createStreamLogReq))
         .map(streamLogMapper::toStreamLogModel)
         .map(ResponseEntity::ok)
         .orElseThrow(() -> new ExceptionType.UnknownException(CONTROLLER_NAME));
```

Comment:
Logs are not my responsibility. It's PR #446 that deals with logs. These changes are necessary because of changes that were submitted to master with PR #457. I'm not too sure about the internal server error.